custom/icds_reports/management/commands/sql_scripts/nos_of_deaths.sql

```diff
@@ -0,0 +1,17 @@
+
+SELECT
+  convert_to(state, 'UTF8') AS state,
+  convert_to(district, 'UTF8') AS district,
+  convert_to(awc, 'UTF8') AS AWC,
+  death_month,
+  count(*) as no_of_deaths
+FROM (
+      SELECT awc_location.state_name AS state,
+             awc_location.district_name AS district,
+             awc_location.awc_name AS awc,
```

Comment:
It looks like the spec wants AWC code not awc name, which I believe are different.

Reply:
Yes that is correct. I intentionally made it because. I see Awc_name in the format of `awc_name  [awc_code]` except the testing aww whose awc_code is also not present. Thought would be usefull to get both. and also it does not required extra efforts.
Feel free to object if you feel awc_code would be better to do here.

Reply:
works for me. If they complain, we can fix it